README.md

```
@@ -1,2 +1,59 @@
+- [.github/PULL_REQUEST_TEMPLATE.md](./.github/PULL_REQUEST_TEMPLATE.md) for the PR checklist.
+- [.github/ISSUE_TEMPLATE](./.github/ISSUE_TEMPLATE) for correction, attribution, and removal requests.
+
+## Copyright
+
+The copyright of each editorial file belongs to its original author, publisher, or contest organizer. This repository organizes and distributes editorial references for educational and archival purposes.
+
+If you are a rights holder and need a correction, attribution update, or removal, please open an issue using the templates in `.github/ISSUE_TEMPLATE`.
+
+## License
+
+<img align="right" src="https://149753425.v2.pressablecdn.com/wp-content/uploads/2009/06/OSIApproved_100X125.png">
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)